Variables/Undefined variable/task.md

```diff
@@ -3,11 +3,11 @@
 Variable names may only contain Latin letters, digits, and/or the underscore 
 character, and they cannot start with a digit. They also cannot be any of the 
 reserved <a href="https://docs.python.org/3/reference/lexical_analysis.html#keywords">keywords</a>.
+
 ### Task
-Check what happens if you use a variable which is not defined yet. Change and run the code in the editor – try to print out an undefined name.  
+Check what happens if you use a variable which is not defined yet. Change and run the code in the editor – try to print out an **undefined** name (i.e. type the name of an undefined variable inside the brackets of the `print` statement).  
 This should cause an exception - a `NameError`.
 
 Note, that "check" command only runs the testing system, and you will not see the output, so you need to run the file.
 
-<div class="hint">Type the name of an undefined variable inside the brackets of the <code>print</code> statement. 
-Note that variable names must start with a letter and can contain only letters, '_', and numbers.</div>
+<div class="hint">for example, try to print <code>var1</code> variable instead of <code>variable</code></div>
```

Comment:
<div class="hint">For example, try to print the <code>var1</code> variable instead of <code>variable</code></div>